src/main/java/io/inversion/cloud/model/Column.java

```diff
@@ -64,6 +66,17 @@ public int compareTo(Column o)
       return 0;
    }
 
+   public boolean equals(Object object)
+   {
+      if (object instanceof Column)
+      {
+         Column column = (Column) object;
+         if ((table != null && table.equals(column.table)) && Utils.equal(name, column.name))
```

Comment:
Could column (the object to compare) ever be null?

Reply:
well.. 'object' can be null but 'column' can't after passing through the instanceof check